src/sentry/runner/commands/cleanup.py

```diff
@@ -16,8 +16,10 @@
 from django.db.models import Model, QuerySet
 from django.utils import timezone
 
+from sentry.db.deletion import BulkDeleteQuery
 from sentry.runner.decorators import log_options
 from sentry.silo.base import SiloLimit, SiloMode
+from sentry.utils.query import RangeQuerySetWrapper
```

Comment:
Eventually I will move all the imports I can here (I won't be able to move the ones within process execution).